checker_test.go

```diff
@@ -130,20 +120,42 @@ want:
 	about:   "Equals: not enough arguments",
 	checker: qt.Equals,
 	expectedCheckFailure: `
-not enough arguments provided to "equals" checker: got 0, want 1
+error:
+  bad check: not enough arguments provided to checker: got 0, want 1
+want:
```

Comment:
This is slightly odd. It looks like we're printing the (non-existent) want argument here. Perhaps "missing" might be better?

Reply:
I'll change it to "got args" and "want args".